packages/indexer-common/src/indexer-management/resolvers/actions.ts

```diff
@@ -246,3 +263,28 @@ export default {
     return await actionManager.executeApprovedActions()
   },
 }
+
+// Helper function to assess equality among a enqueued and a proposed actions
+function compareFailedActions(enqueued: Action, proposed: ActionInput): boolean {
+  // actions are not the same if they target different deployments
```

Comment:
hey tiago👋  wondering if it would be nice to also check `action.source`, as we only filter if the enqueued action is an action made by `indexerAgent`?

Reply:
Resolved in 53a2cb8.

I've opted to keep this check outside of `compareFailedActions` because we are not strictly comparing for equality.